com.gitlab.adnan338.Nixwriter.yaml

```diff
@@ -0,0 +1,47 @@
+id: com.gitlab.adnan338.Nixwriter
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: nixwriter
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=host
```

Comment:
What does it needs host access for?

Reply:
To be able to read `.iso` files from a wider range of directories instead of just `$HOME`.  This is useful when you have a collection of `.iso` images in an external hard-drive and want to create a bootable image out of it without copying it inside home first.

Reply:
The NativeFileChooser should work just fine and without any filesystem permissions needed, just make sure to always the file URI & not the file path. As the later is not the same inside & outside of the sandbox

Reply:
Fixed. Thanks.